### PR TITLE
update envoy-wasm (upstream Envoy v1.16)

### PR DESCRIPTION
**Pull request by yangminzhu**

### CODE REVIEW

**Review by kyessenov — APPROVED** [review]

**Review by PiotrSikora — CHANGES_REQUESTED** [review]
Please update `envoy.bazelrc` (copy of `.bazelrc` from Envoy directory at that SHA).

**Review by kyessenov — APPROVED** [review]

File: extensions/attributegen/plugin_test.cc [hunk 1]
```diff
@@ -187,8 +187,7 @@ class WasmHttpFilterTest : public testing::TestWithParam<TestParams> {
     // vm.
     Extensions::Common::Wasm::createWasm(
         proto_config.config().vm_config(), plugin_, scope_, cluster_manager_,
-        init_manager_, dispatcher_, random_, *api, lifecycle_notifier_,
-        remote_data_provider_,
+        init_manager_, dispatcher_, *api, lifecycle_notifier_, remote_data_provider_,
```
Comment (PiotrSikora) [inline extensions/attributegen/plugin_test.cc:190]:
```
Source file(s) not formatted:
extensions/attributegen/plugin_test.cc
Makefile.core.mk:132: recipe for target 'lint' failed
```

**Review by PiotrSikora — CHANGES_REQUESTED** [review]

File: extensions/attributegen/plugin_test.cc (hunk 1 above)
Comment (yangminzhu) [reply extensions/attributegen/plugin_test.cc]: yeah, thanks, just fixed this.

**Review by PiotrSikora — APPROVED** [review]

**Comment by PiotrSikora** [comment]
@yangminzhu macOS is busted, don't worry about it.

**Pull request merged**